### DISCUSSION TITLE
Continue training after expanding dataset?

### DISCUSSION THREAD

**Discussion by skol101** [coqui-ai/TTS#835] [Archived]
Say I've expanded my speaker's dataset with another hour of speech. The training on a pre-trained model has counted 20k steps so far. 

Does it make sense continue training with a newly expanded dataset or I shall restart training?

**Comment by erogol** [comment]
To me makes sense to continue the training. Even maybe you can train a couple of epoch with only the new dataset to see how it affects the model.
